__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data')
+
+afterAll(() => db.end());
+beforeEach(() => seed(testData))
+
+describe('GET /api/reviews', () => {
+    test('status:200, returns the array of review objects', () => {
+        return request(app)
+            .get('/api/reviews')
+            .expect(200)
+            .then(({body}) => {
+                const { reviews } = body;
+                expect(reviews).toBeInstanceOf(Array);
+                expect(reviews).toHaveLength(13);
```

Comment:
Really like the test for length to ensure there are values in the array before you loop over them checking the objects properties